app/src/main/java/net/gsantner/opoc/ui/FilesystemViewerAdapter.java

```diff
@@ -137,7 +137,7 @@ public void onBindViewHolder(@NonNull ilesystemViewerViewHolder holder, int posi
             holder.title.setTextColor(ContextCompat.getColor(_context, _dopt.accentColor));
         }
 
-        File descriptionFile = file.equals(_currentFolder.getParentFile()) ? fileParent : file;
+        descriptionFile = file.equals(_currentFolder.getParentFile()) ? fileParent : file;
```

Comment:
This is not needed. For selection there is getCurrentSelection()

Reply:
Fix in wladokacur/markor@6c6f39893eb4315466faf99062318f8da95c45a8